src/Header/index.tsx

```diff
@@ -11,8 +14,9 @@ const Header: FC = () => {
         setOpen(true);
     };
 
-    const handleClose = () => {
+    const handleConfirm = (newMovie: MovieInterface) => {
         setOpen(false);
+        store.dispatch(addMovie(newMovie));
```

Comment:
Consider using `react-redux` library. Current approach will work, but eventually you'll get a few problems. Check official docs, please: https://react-redux.js.org/introduction/why-use-react-redux